interface/patient_file/transaction/transactions.php

```diff
@@ -1,104 +1,44 @@
 <?php
-
-
-
-include_once("../../globals.php");
-include_once("$srcdir/transactions.inc");
-require_once("$srcdir/options.inc.php");
-?>
-<html>
-<head>
-    <?php
-    $include_standard_style_js = array("common.js");
-    require_once "{$GLOBALS['srcdir']}/templates/standard_header_template.php";
-    ?>
-
-<script type="text/javascript">
-    // Called by the deleteme.php window on a successful delete.
-    function imdeleted() {
-        top.restoreSession();
-        location.href = 'transaction/transactions.php';
+/**
+ * Patient transactions.
+ *
+ */
+
+
+include_once "../../globals.php";
+include_once "$srcdir/transactions.inc";
+require_once "$srcdir/options.inc.php";
+
+
+$include_standard_style_js = ['common.js'];
+
+/*
+ * This code kind of makes me nervous - it forces in the output of the standard
+ * header into a variable so we can pass it to the twig twmplate.
+ *
+ * I'm nervous because I don't have a lot of experience with ob_* functions.
+ * Ideally, once we get the assets helper class up and running we will use that
+ * instead. See https://stackoverflow.com/questions/2830366/require-once-to-variable
+ * for info on this method - RD
```

Comment:
I agree with this assessment.
Why not make a class now that does the stuff below, so it at least looks nicer here. And then can turn that "rough" class into the fancy class when it's ready?